libvirt/util/expandenv.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var (
```

Comment:
This is not my preferred way of monkey patching, but I did not want to add github.com/bouk/monkey to the project without maintainer approval. @dmacvicar